github-connector/internal/events/sender_test.go

```diff
@@ -59,53 +62,30 @@ func TestSendToKyma(t *testing.T) {
 		assert.Equal(t, expected.Code(), actual.Code())
 	})
 
-	t.Run("should return an internal error when given no event version", func(t *testing.T) {
+	t.Run("should return an internal error when couldn't send a request", func(t *testing.T) {
```

Comment:
You don't need to use mock here. You could pass a regular http.Client{} and specify incorrect service path (e.g. "incorrect.com").